### PR TITLE
SC-13524 Simplify Readme and link to official docs

### PR DESCRIPTION
**Pull request by PabloB94**

### CODE REVIEW

File: charts/sematext-agent/README.md [hunk 1]
```diff
@@ -2,57 +2,24 @@
...
 
-## Introduction
-
 This chart installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
 
 ## Prerequisites
 
 - Kubernetes 1.13+
-- You need to create [a new Docker app in Sematext Cloud](https://apps.sematext.com/ui/integrations/create/docker) to get relevant tokens
+- You need to create [an Infra App in Sematext Cloud](https://apps.sematext.com/ui/monitoring-create/app/infra) to get your infra token.
```
Comment (otisg) [inline charts/sematext-agent/README.md:10]: Add a link to SC EU, too?
infra token ==> Infra App token

File: charts/sematext-agent/README.md [hunk 2]
```diff
@@ -64,49 +31,6 @@ $ helm uninstall st-agent
...
-    sematext/sematext-agent
-```
-
-Alternatively, you can use a YAML file that specifies the values while installing the chart. For example:
-
-```bash
-$ helm repo add sematext https://cdn.sematext.com/helm-charts/
-$ helm install st-agent -f custom_values.yaml sematext/sematext-agent
-```
+For more detailed information, refer to our [official documentation](https://sematext.com/docs/agents/sematext-agent/kubernetes/helm/).
```
Comment (otisg) [inline charts/sematext-agent/README.md:36]: official documentation ==> official Kubernetes Helm chart documentation
(so people don't think we are pointing to top level docs)

**Review by yagogou — COMMENTED** [review]
I can't open the doc url, I assume that has not been deployed yet. But if contains what we deleted looks good

File: charts/sematext-agent/README.md (hunk 2 above)
Comment (PabloB94) [reply charts/sematext-agent/README.md:36]: Done

**Comment by PabloB94** [comment]
> I can't open the doc url, I assume that has not been deployed yet. But if contains what we deleted looks good

The docs this will link to are in this PR: https://github.com/sematext/docs/pull/401

File: charts/sematext-agent/README.md [hunk 3]
```diff
@@ -1,58 +1,25 @@
...
-$ helm repo add sematext https://cdn.sematext.com/helm-charts/
-$ helm install st-agent \
-    --set existingSecret.name=sematext-secret \
-    --set existingSecret.hasInfraToken=true \
-    --set region=US \
-    sematext/sematext-agent
+```sh
+helm install --name sematext-agent \
+  --set infraToken=<YOUR_INFRA_TOKEN> \
+  --set region=<"US" or "EU"> \
```
Comment (otisg) [inline charts/sematext-agent/README.md:19]: No quotes needed from what I can tell - https://sematext.com/docs/agents/sematext-agent/kubernetes/helm/

**Review by otisg — APPROVED** [review]

File: charts/sematext-agent/README.md (hunk 3 above)
Comment (PabloB94) [reply charts/sematext-agent/README.md]: I removed them so it's the same info on both. It works without them

**Pull request merged**